test/Integration/tpp-add.mlir

```diff
@@ -0,0 +1,78 @@
+// RUN: tpp-opt %s -default-tpp-passes="tpp-to-loops" | \
+// RUN: tpp-run -print \
+// RUN:  -e entry -entry-point-result=void | \
+// RUN: FileCheck %s -check-prefix=EXE
+
+// RUN: tpp-opt %s -convert-linalg-to-loops -convert-vector-to-scf -convert-scf-to-cf -expand-strided-metadata -lower-affine -convert-arith-to-llvm -convert-vector-to-llvm -finalize-memref-to-llvm -arith-expand -convert-math-to-llvm -convert-func-to-llvm -canonicalize -reconcile-unrealized-casts | \
```

Comment:
Why are you testing this alternative pipeline? Passing LLVM dialect to `tpp-run` doesn't work, and this doesn't test anything additional than the other two tests.

Reply:
I want to check with Linalg.

Reply:
Ah, right, then remove the llvm passes and it should work.